metricbeat/module/coredns/stats/_meta/fields.yml

```diff
@@ -0,0 +1,90 @@
+- name: stats
+  type: group
+  description: >
+    Contains statistics related to the coreDNS service
+  release: beta
+  fields:
+    - name: panic.count.total
+      type: long
+      description: >
+         Total number of panics
+    - name: dns.request.count.total
+      type: long
+      description: >
+         Total query count
+    - name: dns.request.duration.sec.bucket.*
+      type: object
+      object_type: double
+      description: >
+         Request duration histogram buckets in nanoseconds
+    - name: dns.request.duration.sec.sum
+      type: long
+      format: duration
+      description: >
+         Requests duration, sum of durations in nanoseconds
+    - name: dns.request.duration.sec.count
+      type: long
+      description: >
+         Requests duration, number of requests
+    - name: dns.request.size.bytes.bucket.*
+      type: object
+      object_type: double
+      description: >
+         Request Size histogram buckets
+    - name: dns.request.size.bytes.sum
+      type: double
+      description: >
+        Request Size histogram sum
+    - name: dns.request.size.bytes.count
+      type: double
+      description: >
+        Request Size histogram count
+    - name: dns.request.do.count.total
+      type: long
+      description: >
+         Number of queries that have the DO bit set
+    - name: dns.request.type.count.total
+      type: long
+      description: >
+          Counter of queries per zone and type
+    - name: type
+      type: keyword
+      description: >
+        Holds the query type of the request
+    - name: dns.response.rcode.count.total
+      type: long
+      description: >
+        Counter of responses per zone and rcode
+    - name: rcode
+      type: keyword
+      description: >
+        Holds the rcode of the response
+    - name: family
+      type: keyword
+      description: >
+        The address family of the transport (1 = IP (IP version 4), 2 = IP6 (IP version 6))
+    - name: dns.response.size.bytes.bucket.*
+      type: object
+      object_type: double
+      description: >
+          Response Size histogram buckets
+    - name: dns.response.size.bytes.sum
+      type: double
+      description: >
+        Response Size histogram sum
+    - name: dns.response.size.bytes.count
```

Comment:
Same for all these. I don't like split bytes. In the `_source` we would still have the original value but for querying we would use the long value.

Reply:
:+1: